pkg/linkmode/dynamic.go

```diff
@@ -0,0 +1,8 @@
+//go:build !static_build
+// +build !static_build
+
+package linkmode
```

Comment:
This package does not belong in the `pkg` tree as it makes no sense to be imported by third parties.